### DISCUSSION TITLE
"Failed to set remote answer sdp: Called in wrong state: stable"

### DISCUSSION THREAD

**Discussion by FarJutharat** [stasel/WebRTC-iOS#88] [Q&A]
Error Domain=org.webrtc.RTC_OBJC_TYPE(RTCPeerConnection) Code=-1 "Failed to set remote answer sdp: Called in wrong state: stable" UserInfo={NSLocalizedDescription=Failed to set remote answer sdp: Called in wrong state: stable}

@stasel How to fix this issue? I need remote answer sdp.

**Comment by stasel** [comment]
Him it looks like you are trying to set the remote answer before setting the offer. 
Did you follow the run instructions in the README file?
